analysis/analysis_test.go

```diff
@@ -190,10 +190,15 @@ func TestApplicationAnalysis(t *testing.T) {
 			// Check the analysis issues
 			if len(gotAnalysis.Issues) != len(tc.Analysis.Issues) {
 				t.Errorf("Different amount of issues error. Got %d, expected %d.", len(gotAnalysis.Issues), len(tc.Analysis.Issues))
-				t.Error("Got:")
-				pp.Println(gotAnalysis.Issues)
-				t.Error("Expected:")
-				pp.Println(tc.Analysis.Issues)
+				missing, unexpected := getIssuesDiff(tc.Analysis.Issues, gotAnalysis.Issues)
+				for _, issue := range missing {
+					fmt.Println("Expected issue not found:")
+					pp.Println(issue)
```

Comment:
same as I said about incidents below, I think only the rule ID is enough to keep output concise